File: Linkpoint/src/main/java/com/linkpoint/slproto/mesh/MeshJointTranslations.java
package com.linkpoint.slproto.mesh;

import com.linkpoint.slproto.avatar.SLSkeletonBoneID;
import java.util.EnumMap;

public class MeshJointTranslations {
    public final EnumMap<SLSkeletonBoneID, float[]> jointTranslations = new EnumMap<>(SLSkeletonBoneID.class);
    public float pelvisOffset = 0.0f;
}

File: Linkpoint/src/main/app/com/linkpoint/slproto/mesh/MeshJointTranslations.kt
package com.linkpoint.slproto.mesh

import com.linkpoint.slproto.avatar.SLSkeletonBoneID
import java.util.EnumMap

class MeshJointTranslations {
    val jointTranslations: EnumMap<SLSkeletonBoneID, FloatArray> = EnumMap(SLSkeletonBoneID::class.java)
    var pelvisOffset: Float = 0.0f
}